xmonad-extras.cabal

```diff
@@ -51,7 +51,7 @@ library
     --     ghc-options:    -Werror
 
     if flag(with_sound)
-        build-depends: alsa-mixer >= 0.2
+        build-depends: alsa-mixer >= 0.3
```

Comment:
You need to make the same change in `stack.yaml` to fix the continuous build on Travis-CI.